cookiecutter.json

```
@@ -0,0 +1,15 @@
+{
+    "project_name": "My Site",
+    "project_shortname": "{{ cookiecutter.project_name | slugify() }}",
+    "project_site": "{{ cookiecutter.project_shortname }}.com",
+    "github_repo": "{{ cookiecutter.project_shortname }}/{{ cookiecutter.project_shortname }}",
+    "description": "{{ cookiecutter.project_name }} InvenioILS Instance",
+    "author_name": "CERN",
+    "author_email": "info@{{ cookiecutter.project_site }}",
+    "year": "{% now 'local', '%Y' %}",
+    "python_version": ["3.7", "3.8", "3.9"],
+    "database": ["postgresql"],
+    "search": ["elasticsearch7", "opensearch1"],
```

For now, we did not upgrade to OpenSearch yet :(
```suggestion
    "search": ["elasticsearch7"],
```